include/rtc/description.hpp

```diff
@@ -29,14 +29,23 @@
 
 namespace rtc {
 
+
 class Description {
 public:
-	enum class Type { Unspec = 0, Offer = 1, Answer = 2 };
-	enum class Role { ActPass = 0, Passive = 1, Active = 2 };
 
-	Description(const string &sdp, const string &typeString = "");
+    enum class Type { Unspec = 0, Offer = 1, Answer = 2 };
+	enum class Role { ActPass = 0, Passive = 1, Active = 2 };
+    enum Direction {
+        SEND_ONLY,
```

Comment:
For consistency with the other enums these could be named `SendOnly`, `RecvOnly`, `Both`, and `Unknown`.